server/utils.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/kubernetes-incubator/cri-o/lib/sandbox"
 	"github.com/kubernetes-incubator/cri-o/server/metrics"
 	"github.com/opencontainers/image-spec/specs-go/v1"
-	"github.com/opencontainers/runtime-tools/validate"
```

Comment:
I'd rather push this change up to runtime-tools (which aspires to compile and validate on Windows as well).  If we make `LastCap` platform-agnostic there, all runtime-tools consumers will benefit.

Reply:
didn't i open a PR over there as well?

Reply:
https://github.com/opencontainers/runtime-tools/pull/560 is what I was thinking of

Reply:
i dunno, this is really on the caller to do the right thing

Reply:
> opencontainers/runtime-tools#560 is what I was thinking of

That doesn't look like it's touching `LastCap`.  Maybe it should?

> i dunno, this is really on the caller to do the right thing

In this PR, you have “the right thing” on Windows as `capability.Cap(-1)`.  That seems right to me, and not CRI-O-specific.  Are there cases where it would not be the right thing?

Reply:
hmm. dunno really. @rhatdan @mrunalp @runcom opinion?

Reply:
https://github.com/opencontainers/runtime-tools/pull/588 is the alternative

Reply:
I like that alternative.

Reply:
right on!